### DISCUSSION TITLE
why red1 classname is showing while is not used?

### DISCUSSION THREAD

**Discussion by naveennsit** [FullHuman/purgecss#575] [Q&A] [answered]
could you please tell me why `red1` classname  is showing while it is not used ? I am using this plugin https://www.npmjs.com/package/purgecss to remove unused css .

I am following this documentation 
https://purgecss.com/configuration.html
here is my code

https://codesandbox.io/s/priceless-mclaren-8tcjl?file=/pages/index.js

    const xx = await new PurgeCSS().purge({
        content: ["**/*", "pages/**/*.{js,jsx,ts,tsx}"],
        css: [
          {
            raw: ".red1 { color: red }"
          }
        ]
      });
      console.log("xx", xx);

In my code I didn't used `red1` class in my project directory.but still in console is showing why ?

**Comment by Ffloriel** [comment, marked as answer]
From what I see, there is an issue with how PurgeCSS is called. The configuration is asking to look at pages/** files but you are calling PurgeCSS with the CSS in index.js.
PurgeCSS will essentially look at index.js to determine what CSS is used, and will see in this case `red1` from
```
const xx = await new PurgeCSS().purge({
    content: ["**/*", "pages/**/*.{js,jsx,ts,tsx}"],
    css: [
      {
        raw: ".red1 { color: red }" // This line has red1, so PurgeCSS will keep it
      }
    ]
  });
```
